codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/MainSourceGeneratorSuite.scala

```diff
@@ -182,6 +183,7 @@ class MainSourceGeneratorSuite extends munit.FunSuite {
         |import com.example.service.view.MyServiceViewImpl;
         |import com.example.service.view.MyServiceViewProvider;
         |import com.example.service.view.ServiceOuterClass;
+        |
         |import java.util.function.Function;
```

Comment:
shall we have the std lib imports at the top?

Reply:
I agree that seems nicer, but I tried to follow the IntelliJ default conventions which seem to put them at the bottom...

Reply:
I'm not sure that this has any added value for us (and for the user). 

A simple alphabetical sort should be enough for generated code. 

And for the generated code that can be edited by the user, we should not care because their IDE will sort it according to their preferences.

Reply:
I agree the exact ordering is not super important but having that grouping is quite nice.